src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb

```diff
@@ -33,6 +33,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
         End Function
 
         <Extension()>
+        Public Function IsPreProcessorKeywordContext(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
+            Return IsPreProcessorKeywordContext(
```

Comment:
The overload of this method checks for syntax kind and calls `GetPreviousTokenIfTouchingWord` which has been implemented separately for VB and C#.